src/components/Combobox/Combobox.stories.tsx

```diff
@@ -153,3 +150,30 @@ export const CustomComponents = () => (
     }}
   />
 )
+
+export const MultiSelect = () => (
+  <Combobox<Fruit>
+    itemIsEqual={(a, b) => a.label === b.label}
+    value={fruits.filter(
+      (e) =>
+        optionsKnob('values', Object.fromEntries(fruits.map((e) => [e.label, e.label])), [], {
```

Comment:
I went to the branch's deployment to see how it works and.. it doesn't seem to be working :sweat_smile:

Reply:
You're right, it wasn't working, but now it does :grin: